src/renderers/shared/ReactDebugTool.js

```diff
@@ -51,11 +109,11 @@ var isProfiling = false;
 var flushHistory = [];
 var lifeCycleTimerStack = [];
 var currentFlushNesting = 0;
-var currentFlushMeasurements = null;
-var currentFlushStartTime = null;
+var currentFlushMeasurements = [];
```

Comment:
I'm not very happy with this change but I can't think of a better way that wouldn't involve a massive change.

https://www.facebook.com/groups/flowtype/permalink/1132169456831668/